packages/evm/contracts/registry/GuardiansRegistry.sol

```diff
@@ -689,15 +682,15 @@ contract GuardiansRegistry is ControlledRecoverable, IGuardiansRegistry, IERC900
         }
 
         _updateAvailableBalanceOf(_guardian, amountToActivate, false);
-        emit GuardianActivated(_guardian, nextTermId, amountToActivate, _sender);
+        emit GuardianActivated(_guardian, nextTermId, amountToActivate, msg.sender);
```

Comment:
Would agree that this last `sender` event param seems pretty unnecessary.

Reply:
Totally